tests/test_utils.py

```diff
@@ -24,6 +25,11 @@ def my_func():
                 my_func()
             self.assertEqual(count['count'], expected_attempts)
 
+    def test_timestamps(self):
+        dt = datetime.utcnow()
+        ts = timestamp(dt)
+        self.assertEqual(datetime_from_timestamp(ts), dt)
+        timestamp_now()
```

Comment:
Should this line have an assertion statement?

Reply:
Yeah... I wasn't sure how to test this so I figured calling it is better than nothing.

Reply:
assertion added!